docs/sample_config.yaml

```diff
@@ -75,6 +75,11 @@ pid_file: DATADIR/homeserver.pid
 #
 #public_baseurl: https://example.com/
 
+# The address (and optional port) to direct federation traffic to.
+# Setting this option will enable .well-known delegation.
```

Comment:
well, this isn't really true, because there are a whole bunch of other steps involved in enabling .well-known delegation - like making sure that traffic to the .well-known URI actually reaches synapse.